src/robusta/cli/self_host.py

```diff
@@ -0,0 +1,180 @@
+    )
+
+
+def write_values_files(
+    values_path: str,
+    backendconfig_path: str,
+    values: dict[str, Any],
+    backendProfile: BackendProfile,
+):
+    with open(values_path, "w") as output_file:
+        yaml.safe_dump(values, output_file, sort_keys=False)
+        typer.secho(
+            f"Saved configuration to {values_path} - save this file for future use!",
+            fg="red",
+        )
+
+    with open(backendconfig_path, "w") as output_file:
+        json.dump(
+            backendProfile.dict(exclude={"custom_profile"}), output_file, indent=1
+        )
+        typer.secho(
+            f"Saved configuration to {backendconfig_path} - save this file for future use!",
+            fg="red",
+        )
+
+
+class RobustaUI(BaseModel):
+    RELAY_HOST: str = ""
+    SUPABASE_URL: str = ""
+    SUPABASE_KEY: str = ""
+    service = {"nodePort": 30311}  # platform.domain
+
+    def __init__(self, domain: str, anon_key: str):
+        super().__init__(
+            RELAY_HOST=f"https://api.{domain}",
+            SUPABASE_URL=f"https://db.{domain}",
+            SUPABASE_KEY=anon_key,
+        )
+
+
+class RobustaRelay(BaseModel):
+    domain: str = ""
+    storePassword: str = ""
+    storeUser: str = "apiuser-robustarelay@robusta.dev"
+    storeUrl: str = ""
+    storeApiKey: str = ""  # anon key
+    slackClientId: str = "your-client-id"
+    slackClientSecret: str = "your-client-secret"
+    slackSigningSecret: str = "your-signing-secret"
+    syncActionAllowedOrigins: str = ""
+    provider: str = ""
+    apiNodePort: int = 30313  # api.domain
+    wsNodePort: int = 30314  # relay.domain
+
+    def __init__(self, domain: str, anon_key: str, provider: str, storePW: str):
+        super().__init__(
+            domain=domain,
+            storeUrl=f"https://db.{domain}",
+            syncActionAllowedOrigins=f"https://platform.{domain}",
+            storeApiKey=anon_key,
+            provider=provider,
+            storePassword=storePW,
+        )
+
+
+class SelfHostValues(BaseModel):
+    STATIC_IP_NAME: str = "robusta-platform-ip"
+    RELAY_PASSWORD: str = gen_secret(12)
+    RELAY_USER: str = "apiuser-robustarelay@robusta.dev"
+    DOMAIN: str = ""
+    PROVIDER: str = ""
+    # SUPABASE
+    JWT_SECRET: str = gen_secret(32)
+    JWT_EXPIRY: int = 3600
+    ANON_KEY: str = JWT.encode(
+        {"role": "anon", "iss": "supabase", "iat": 1661029200, "exp": 1818795600},
+        JWT_SECRET,
+    )
+    SERVICE_ROLE_KEY: str = JWT.encode(
+        {
+            "role": "service_role",
+            "iss": "supabase",
+            "iat": 1661029200,
+            "exp": 1818795600,
+        },
+        JWT_SECRET,
+    )
+    SUPABASE_URL: str = "http://kong:8000"  # Internal URL
+    PUBLIC_REST_URL: str = ""  ## Studio Public REST endpoint - replace this if you intend to use Studio outside of localhost
+
+    # POSTGRES
+    POSTGRES_PORT: int = 5432
+    POSTGRES_STORAGE: str = "50Gi"
```

Comment:
Not sure if we decided 50Gi
Maybe using 100Gi is a better default

Do we also want to expose the `storageClass`?

Reply:
updated to 100 and added the storage class as well